src/Umbraco.Infrastructure/BackgroundJobs/Jobs/WebhookFiring.cs

```diff
@@ -90,20 +88,30 @@ public async Task RunJobAsync()
 
     private async Task<HttpResponseMessage?> SendRequestAsync(IWebhook webhook, string eventName, string? serializedObject, int retryCount, CancellationToken cancellationToken)
     {
-        using var httpClient = new HttpClient();
-
-        var stringContent = new StringContent(serializedObject ?? string.Empty, Encoding.UTF8, MediaTypeNames.Application.Json);
-        stringContent.Headers.TryAddWithoutValidation("Umb-Webhook-Event", eventName);
+        using HttpClient httpClient = _httpClientFactory.CreateClient(Constants.HttpClients.WebhookFiring);
 
-        foreach (KeyValuePair<string, string> header in webhook.Headers)
+        var request = new HttpRequestMessage(HttpMethod.Post, webhook.Url)
         {
-            stringContent.Headers.TryAddWithoutValidation(header.Key, header.Value);
-        }
+            Version = httpClient.DefaultRequestVersion,
+            VersionPolicy = httpClient.DefaultVersionPolicy,
+        };
 
         HttpResponseMessage? response = null;
         try
         {
-            response = await httpClient.PostAsync(webhook.Url, stringContent, cancellationToken);
+            // Add headers
+            request.Headers.Add("Umb-Webhook-Event", eventName);
```

Comment:
To aid the contextual awareness of a webhook execution (see https://github.com/umbraco/Umbraco-CMS/discussions/15238) it would be good to include the retry count here as well:

```suggestion
            request.Headers.Add("Umb-Webhook-Event", eventName);
            request.Headers.Add("Umb-Webhook-RetryCount", retryCount.ToString());
```